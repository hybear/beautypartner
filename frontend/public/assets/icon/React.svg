<svg width="76" height="67" viewBox="0 0 76 67" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M0.224243 33.4271C0.224243 39.0246 5.79427 44.2497 14.5755 47.1951C12.8219 55.8576 14.2614 62.8399 19.0688 65.5603C24.01 68.3569 31.3716 66.4212 38.3589 60.4276C45.1698 66.1791 52.0532 68.4425 56.8564 65.7142C61.7932 62.9098 63.2944 55.4541 61.4925 46.521C70.6388 43.5538 75.7549 39.0983 75.7549 33.4271C75.7549 27.9597 70.1748 23.2297 61.5118 20.3033C63.4518 11.0011 61.808 4.18834 56.8015 1.35541C51.9748 -1.37586 45.0113 1.00468 38.1003 6.9065C30.9103 0.642294 24.0157 -1.27012 19.0144 1.57068C14.1922 4.30982 12.8163 11.4191 14.5755 20.2418C6.09054 23.1447 0.224243 27.9818 0.224243 33.4271Z" fill="white"/>
<path d="M59.6772 23.2093C58.9633 22.969 58.2432 22.7463 57.5178 22.5416C57.6381 22.062 57.7494 21.5801 57.8515 21.0965C59.486 13.3105 58.4173 7.0381 54.7677 4.97334C51.2686 2.99305 45.5456 5.05781 39.7657 9.99338C39.1962 10.4808 38.6393 10.9823 38.0956 11.4974C37.7315 11.1549 37.3609 10.8191 36.9841 10.4901C30.9266 5.21221 24.8549 2.98839 21.2091 5.05927C17.7132 7.04509 16.6778 12.942 18.1493 20.3207C18.2957 21.0514 18.461 21.7784 18.6451 22.5008C17.786 22.7403 16.9562 22.9955 16.1624 23.267C9.06087 25.6964 4.52539 29.5044 4.52539 33.4539C4.52539 37.5333 9.39395 41.625 16.7912 44.1057C17.391 44.3059 17.9958 44.4914 18.605 44.6621C18.4071 45.4404 18.2314 46.2241 18.0781 47.012C16.6751 54.2626 17.7708 60.0197 21.2575 61.9932C24.8587 64.0309 30.9029 61.9364 36.7885 56.8881C37.2537 56.4891 37.7206 56.0658 38.1882 55.6221C38.7771 56.1797 39.3818 56.7211 40.0014 57.2456C45.7024 62.0594 51.3327 64.0032 54.8164 62.0244C58.4144 59.9806 59.5834 53.7959 58.0653 46.2713C57.9495 45.6965 57.8144 45.1095 57.663 44.5118C58.0875 44.3889 58.504 44.2618 58.9104 44.1296C66.6004 41.6296 71.6034 37.5878 71.6034 33.4539C71.6034 29.4901 66.9219 25.6565 59.6772 23.2093Z" fill="#53C1DE"/>
<path d="M58.009 41.461C57.6424 41.5802 57.2659 41.6952 56.8824 41.8068C56.0334 39.1696 54.8875 36.3653 53.4857 33.4682C54.8237 30.6397 55.925 27.8714 56.7503 25.2511C57.4363 25.4463 58.1025 25.6517 58.7443 25.8687C64.9517 27.9652 68.7379 31.0656 68.7379 33.4542C68.7379 35.9988 64.6486 39.3022 58.0093 41.461H58.009ZM55.2541 46.8178C55.9253 50.1451 56.0212 53.1534 55.5765 55.5051C55.1769 57.6182 54.3736 59.027 53.3803 59.5912C51.2664 60.792 46.7454 59.2312 41.8704 55.1145C41.3114 54.6426 40.7482 54.1386 40.1836 53.6055C42.0737 51.5769 43.9627 49.2185 45.8062 46.5997C49.0491 46.3174 52.1127 45.8557 54.891 45.2256C55.0276 45.7674 55.1493 46.2984 55.2541 46.8178ZM27.3936 59.3838C25.3283 60.0996 23.6834 60.12 22.6892 59.5574C20.5732 58.3602 19.6939 53.7372 20.8935 47.5368C21.0362 46.804 21.1995 46.0753 21.3833 45.3514C24.1308 45.9477 27.1721 46.3768 30.4228 46.6355C32.2785 49.1981 34.222 51.5536 36.1807 53.6163C35.7644 54.0114 35.3383 54.3965 34.9027 54.771C32.3001 57.0036 29.6922 58.5871 27.3939 59.3838H27.3936ZM17.718 41.4453C14.4472 40.3483 11.746 38.9226 9.89448 37.367C8.23087 35.9688 7.39075 34.5807 7.39075 33.4542C7.39075 31.0577 11.0324 28.0004 17.1059 25.9226C17.8682 25.6629 18.6381 25.4252 19.4145 25.2098C20.2541 27.8889 21.3551 30.6901 22.6839 33.5227C21.3379 36.3976 20.2211 39.2433 19.3739 41.9539C18.8177 41.7977 18.2656 41.6281 17.718 41.4453ZM20.9615 19.781C19.7007 13.4588 20.5379 8.68988 22.6447 7.4932C24.889 6.21845 29.851 8.03619 35.0811 12.5931C35.4154 12.8844 35.7512 13.1894 36.0875 13.5048C34.1389 15.5583 32.2132 17.896 30.3735 20.4437C27.2188 20.7307 24.1988 21.1915 21.4225 21.8088C21.2515 21.1367 21.0978 20.4604 20.9615 19.7807V19.781ZM49.8957 26.7918C49.24 25.6803 48.5599 24.5828 47.856 23.5001C49.9964 23.7652 52.0468 24.1177 53.9723 24.5491C53.3943 26.3671 52.6738 28.2682 51.8253 30.217C51.2054 29.0628 50.5621 27.9208 49.8957 26.7918ZM38.0964 15.5143C39.418 16.9192 40.7417 18.4882 42.0437 20.1903C39.4026 20.0678 36.757 20.0672 34.1157 20.1885C35.419 18.5022 36.754 16.9355 38.0964 15.5143ZM26.2213 26.8111C25.5633 27.9311 24.932 29.066 24.3279 30.2149C23.4934 28.2728 22.7795 26.3633 22.1964 24.5194C24.1103 24.099 26.1512 23.7556 28.2777 23.4948C27.5658 24.5843 26.8802 25.69 26.2213 26.8111ZM28.3388 43.6144C26.142 43.3737 24.0702 43.0478 22.157 42.6391C22.7492 40.7625 23.4789 38.8119 24.3312 36.8281C24.9378 37.9779 25.5719 39.1135 26.2329 40.2341C26.9074 41.3771 27.6096 42.5042 28.3388 43.6144ZM38.1756 51.5929C36.8175 50.1547 35.4629 48.5642 34.1398 46.8534C35.424 46.9029 36.7335 46.9279 38.0643 46.9279C39.4317 46.9279 40.7833 46.8979 42.1135 46.8397C40.8073 48.5817 39.4884 50.1745 38.1756 51.5929ZM51.8527 36.7271C52.7492 38.7324 53.505 40.6725 54.1052 42.5173C52.1602 42.9528 50.0605 43.3035 47.848 43.564C48.557 42.461 49.2436 41.3443 49.9073 40.2145C50.5821 39.0664 51.2307 37.9036 51.8527 36.7271ZM47.425 38.8099C46.4089 40.542 45.3356 42.2412 44.207 43.9048C42.2168 44.0443 40.1602 44.1163 38.0643 44.1163C35.9771 44.1163 33.9465 44.0525 31.9911 43.9281C30.8332 42.2683 29.7396 40.5662 28.7129 38.8253C27.6909 37.0937 26.7363 35.3246 25.8511 33.5218C26.734 31.7185 27.6857 29.9485 28.7043 28.2154V28.216C29.7224 26.4826 30.8074 24.7877 31.9567 23.1351C33.9516 22.9871 35.997 22.9096 38.0646 22.9096C40.1415 22.9096 42.1895 22.9877 44.1838 23.1368C45.3215 24.7869 46.3995 26.4759 47.4158 28.2006C48.4378 29.9298 49.4022 31.6912 50.3075 33.4819C49.4115 35.2909 48.4502 37.068 47.4253 38.8099H47.425ZM53.3364 7.40959C55.5824 8.68085 56.4561 13.8069 55.0448 20.5294C54.9523 20.9673 54.8518 21.4035 54.7432 21.8379C51.9604 21.2078 48.9384 20.7388 45.7744 20.4475C43.9312 17.8721 42.0215 15.5303 40.1043 13.5034C40.6067 13.0281 41.121 12.5652 41.6468 12.115C46.5991 7.88588 51.2275 6.21612 53.3364 7.40959Z" fill="white"/>
<path d="M38.0644 27.5762C41.3726 27.5762 44.0542 30.2078 44.0542 33.4538C44.0542 36.7001 41.3726 39.3317 38.0644 39.3317C34.7564 39.3317 32.0746 36.7001 32.0746 33.4538C32.0746 30.2078 34.7564 27.5762 38.0644 27.5762Z" fill="#53C1DE"/>
</svg>
