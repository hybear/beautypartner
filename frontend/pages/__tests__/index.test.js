import React from 'react';
import { shallow, mount } from 'enzyme';
import Index from '../index';

describe('index page', () => {
  it('should have SignUp component', () => {
    const wrapper = shallow(<Index />);

    expect(wrapper.find('Canvas')).toHaveLength(1);
  });
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<Index />);

    expect(wrapper).toMatchSnapshot();
  });
});
