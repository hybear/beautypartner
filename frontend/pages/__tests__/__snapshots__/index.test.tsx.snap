// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index page renders and matches the snapshot 1`] = `<SignUp />`;
